test/integration/user/api/interactivity.test.js

```diff
@@ -225,15 +225,15 @@ describe('Interactivity', () => {
                 });
 
                 it('should not fire a featureEnter event when the mouse is moved inside the same feature', done => {
-                    const featureClickOutSpy = jasmine.createSpy('featureClickOutSpy');
+                    const featureEnterSpy = jasmine.createSpy('featureEnterSpy');
                     onLoaded(() => {
-                        // Move mouse inside a feature 1
-                        util.simulateMove({ lng: 5, lat: 5 });
-                        interactivity.on('featureEnter', featureClickOutSpy);
+                        interactivity.on('featureEnter', featureEnterSpy);
                         interactivity.on('featureHover', () => {
-                            expect(featureClickOutSpy).not.toHaveBeenCalled();
+                            expect(featureEnterSpy).toHaveBeenCalledTimes(1);
                             done();
                         });
+                        // Move mouse inside a feature 1
+                        util.simulateMove({ lng: 5, lat: 5 });
                         // Move mouse inside the same feature 1
                         util.simulateMove({ lng: 5, lat: 15 });
```

Comment:
we never get here since `done` is called after the first `simulateMove` I think we should call the first simulateMove  before setting the featureHover callback

Reply:
Yes :+1:

Reply:
fixed